.github/ISSUE_TEMPLATE/submit-method-package.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Submit method package
+about: Form to submit a new or updated method package
+
+---
+
+Thank you for submitting a method package to the HCA DCP Methods Registry!
+
+To expedite your package's addition to https://data.humancellatlas.org/analyze/methods,
+please provide the following package metadata.
+
+Required:
+- Tool title: 
+- Contact name: 
+- Contact email: 
+- Who to attribute: 
+- Language: 
+- GitHub repository URL: (_e.g. https://github.com/yourusername/yourpackagename_)
+- Upstream registry URL: (_e.g. https://bioconductor.org/packages/release/bioc/html/slingshot.html_)
+- 2-3 sentence description:
```

Comment:
Should we note that descriptions >150 characters will be truncated to 150 char on the short listing page with full description on the details page?

Reply:
Done.